src/gridcoin/voting/payloads.h

```diff
@@ -169,6 +169,13 @@ class PollPayload : public IContractPayload
     {
         READWRITE(m_version);
         READWRITE(m_poll);
+
+        // The poll m_addditional_fields is serialized here rather than in the poll class, because it depends on the
+        // payload version.
```

Comment:
It'd be nice to have the same comment in the poll class for clarity.

Reply:
Good point.